api4/system.go

```diff
@@ -68,6 +69,7 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.System.Handle("/notices/{team_id:[A-Za-z0-9]+}", api.APISessionRequired(getProductNotices)).Methods("GET")
 	api.BaseRoutes.System.Handle("/notices/view", api.APISessionRequired(updateViewedProductNotices)).Methods("PUT")
 	api.BaseRoutes.System.Handle("/support_packet", api.APISessionRequired(generateSupportPacket)).Methods("GET")
+	api.BaseRoutes.System.Handle("/first_admin_setup", api.APISessionRequired(getFirstAdminCompleteSetup)).Methods("GET")
 	api.BaseRoutes.System.Handle("/onboarding/complete", api.APIHandler(completeOnboarding)).Methods("POST")
```

Comment:
I forgot to add the session handler as part of my PR. Mind adding it here?

```suggestion
	api.BaseRoutes.System.Handle("/first_admin_setup", api.APISessionRequired(getFirstAdminCompleteSetup)).Methods("GET")
	api.BaseRoutes.System.APISessionRequired("/onboarding/complete", api.APIHandler(completeOnboarding)).Methods("POST")
```